sys/abi/src/lib.rs

```diff
@@ -386,6 +386,10 @@ pub enum UsageError {
     BadKernelMessage,
     BadReplyFaultReason,
     NotSupervisor,
+
+    // A server is attempting to reply with a message that is too large for the
+    // client to handle.
```

Comment:
nit: should these be doc comments? the others all are
```suggestion
    /// A server is attempting to reply with a message that is too large for the
    /// client to handle.
```